src/main/drivers/rangefinder/rangefinder_lidarmt.c

```diff
@@ -67,7 +66,7 @@ static int32_t mtRangefinderGetDistance(rangefinderDev_t * dev) {
     UNUSED(dev);
     if (hasNewData) {
         hasNewData = false;
-        return (sensorData >= 0) ? sensorData : RANGEFINDER_OUT_OF_RANGE;
+        return (sensorData > 2.0) ? sensorData : RANGEFINDER_OUT_OF_RANGE; // MTF lidars has 2cm deadzone
```

Comment:
Does MTF lidar return value smaller than 3? And even when such value is returned, max error would be smaller than 2 cm, negligible ni real world,

Reply:
I tried it, and it returns values of less than 3. I made this modification only to follow the lidar's datasheet.